frontend/class-twitter.php

```diff
@@ -229,7 +229,7 @@ private function taxonomy_description() {
 			return $meta_desc;
 		}
 
-		return trim( strip_tags( term_description() ) );
+		return trim( wp_strip_all_tags( term_description() ) );
```

Comment:
As `wp_strip_all_tags` already trims the content, this does not have to be wrapped again.